demystifying_sql/demystifying_sql.md

```diff
@@ -0,0 +1,312 @@
+| TRSH789    | 8837498101   |  2020-11-29    | 0      |
+| TRSH789    | 8871386401   |  2021-04-01    | 0      |
+| SMLE321    | 8901861569   |  2021-11-22    | 1      |
+| ABC123     | 8927551899   |  2021-12-30    | 0      |
+| ABC123     | 8954998113   |  2022-03-19    | 1      |
+
+
+<h4>A Sample `medication_order` table</h4>
+
+<!-- data-type="none" -->
+| patient_id | provider_id | med_id | order_date |
+| -------    | -----------   | --- | ------    | ----- |
+| ABC123     | 491272    | 8000412   | 2021-05-15 |  
+| ABC123     | 491272  | 7960004   | 2022-02-01 |
+| SMLE321    | 223618    | 8000412 | 2020-08-19 |
+
+The primary benefit of the **relational database** model is the ability to use columns containing the same data (things like patient IDs) to create complex reports combining information from multiple tables.  This enables users of the data to derive specific information from the data in highly customizable ways.
+
+Now that you have that background, you can think of SQL as the computer code (the "Language" in Structured Query Language) that you can use to ask explicit questions (the "Query" in SQL) about the information in your Relational Database.
+
+<div class = "hint">
+
+Where did SQL come from?  SQL was created in the early 1970's by IBM as a method for more easily accessing information from their internal database system.
+
+By 1979 Relational Software, Inc. (now Oracle Corporation) released the first commercially available implementation of SQL as a part of their Oracle V2 database application.
+
+Today SQL is the most common programing language for extracting and organizing data in relational database dystems.
+
+</div>
+
+
+### Quiz: SQL and Relational Databases
+
+<div class = "question">
+
+What does "SQL" stand for?
+
+[[ ]] Sequential Query Language
+[[X]] Structured Query Language
+[[ ]] Simple Question Language
+[[ ]] Syntactic Question Language
+[[ ]] Structured Questing Language
+*********
+
+SQL, which can be pronounced "sql" or S-Q-L, stands for Structured Query Language!
+
+***********
+</div>
+
+<div class = "question">
+
+Which of these correctly describe relational databases?  Choose all the correct options.
+
+[[ ]] Relational databases are an alternative to SQL
+[[X]] Relational databases organize data into tables
+[[ ]] Tables in relational databases are organized into ranks and files
+[[X]] Relational databases permit the extraction of highly specific data
+[[X]] The "relational" in "relational database" refers to data that occurs in multiple places and allows for linking, or "relating" data
+*********
+
+Relational databases are a data storage solution and they store data in tables, which are in turn organized into rows and columns.  Relational databases use SQL as a way to access data, search for data meeting particular conditions, and connect related data, using columns that tables have in common.  This allows for extracting data in very specific and customized way.
```

Comment:
There is a class for answer explanations (it's "answer" I think), which I think will make the explanations stand out a bit more.

Reply:
good call, added